scripts/update_configs_test.py

```diff
@@ -193,8 +235,8 @@ def mock_getpass(prompt):
             'since it is used in\n'
             '# the existing storage models for UserStatsModel.\n'
             'DASHBOARD_STATS_DATETIME_STRING_FORMAT = \'%Y-%m-%d\'\n')
-        expected_feconf_text = '%sMAILGUN_API_KEY = \'%s\'\n' % (
-            feconf_text, mailgun_api_key)
+        expected_feconf_text = feconf_text.replace('None', '\'%s\'' % (
```

Comment:
Don't do this -- instead, hardcode the explicit text.

Otherwise, let's say someone adds a key below MAILGUN_API_KEY that also has value None, and goes ahead and updates this block of code as well without reading the surrounding code. Then the expected value will be wrong.